lib/tasks/verify-dot-npmignore.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const process = require('process');
+const readline = require('readline');
+const { createReadStream } = require('fs');
+const { fileExists} = require('../helpers/files');
+const path = require('path');
+const isCI = require('is-ci');
+
+async function dotNpmignore(config) {
+	const result = [];
+	const npmignorePath = path.join(config.cwd, '/.npmignore');
+	const exists = await fileExists(npmignorePath);
+
+	if (exists) {
+		const stream = createReadStream(npmignorePath);
+		const reader = readline.createInterface({
+			input: stream
+		});
+
+		let lineNumber = 0;
+		for await (const line of reader) {
+			const match = line.trim().match(/(main\.scss|main\.js|src\/?|demos\/?)/);
+			if (match) {
```

Comment:
I think this is the first `for await` I've seen in any of our projects :D

Reply:
it seems to have upset eslint: https://github.com/Financial-Times/origami-build-tools/pull/1039/checks?check_run_id=3420607722

Reply:
i've bumped our ecmaVersion to 2019